MANIFEST.in

```diff
@@ -0,0 +1,35 @@
+include *.txt
```

Comment:
Could you change this to CMakeLists.txt?

Reply:
```suggestion
include CMakeLists.txt
```

Reply:
Sure, done. Though in general, if you did add a something-else.txt file, I think I'd expect it in the SDist? .txt files usually aren't large, and generally SDists are supposed to basically be the same thing as git archives, unless you have large files.